xcm/src/v0/multi_asset.rs

```diff
@@ -290,53 +294,64 @@ impl MultiAsset {
 	}
 }
 
-impl From<MultiAsset1> for MultiAsset {
-	fn from(a: MultiAsset1) -> MultiAsset {
+impl TryFrom<MultiAsset1> for MultiAsset {
```

Comment:
opinionated suggestion: `v1::Foo` cleaner than needing a manual import and `Foo1`.